app/views/publish/courses/engineers_teach_physics/new.html.erb

```diff
@@ -0,0 +1,61 @@
+<% content_for :page_title, title_with_error_prefix("Engineers Teach Physics", @errors && @errors.any?) %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(@back_link_path) %>
+<% end %>
+
+<%= render "publish/shared/errors" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render CaptionText::View.new(text: t("course.add_course")) %>
+
+    <%= form_with url: continue_publish_provider_recruitment_cycle_courses_engineers_teach_physics_path(
+                    @provider.provider_code,
+                    @provider.recruitment_cycle_year,
+                  ),
+                  method: :get do |form| %>
+
+      <%= render "publish/courses/new_fields_holder", form:, except_keys: [:engineers_teach_physics_query] do |fields| %>
+
+        <%#= render "form_fields", form: fields %>
+        <%= render "publish/shared/error_wrapper", error_keys: [:engineers_teach_physics_query], data_qa: "course__engineers_teach_physics" do %>
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+              <h1 class="govuk-fieldset__heading">
+                Engineers Teach Physics
+              </h1>
+            </legend>
+            <p class="govuk-body">
+              Engineers Teach Physics is a new campaign to help engineers become physics teachers.
+            </p>
+            <p class="govuk-body">
+              For more information, email <%= govuk_mail_to("itt.engagement@education.gov.uk", subject: "Engineers Teach Physics") %>
```

Comment:
Can we stick the email in to settings please

Reply:
Good idea